### PR TITLE
Confirm support for Django 1.10

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-    py{27,34,35}-django{18,19},
-    py{27,34,35}-django-master,
+    py{27,34,35}-django{18,19,110,-master},
 
 [testenv]
 commands = {envpython} runtests.py
 deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
+    django110: Django>=1.10a1,<1.11
```
Comment (edmorley) [inline tox.ini]: This can now be updated to `Django>=1.10,<1.11` :-)

Comment (timgraham) [inline tox.ini]: Yes, although it could be helpful to leave this pattern so it's obvious how to add support for the next prerelease of Django.

**Comment by edmorley** [comment]
@jwhitlock I don't suppose you could merge this? :-)

**Comment by jwhitlock** [comment]
@edmorley yes, it's been merge ready for a while. thanks for the ping.

**Comment by edmorley** [comment]
Many thanks!

**Pull request merged**